flightmate/src/main/java/com/flightmate/libs/services/SessionService.java

```diff
@@ -6,22 +6,31 @@
 import com.flightmate.beans.User;
 
 public class SessionService {
-	public static SessionService srv = new SessionService();
-	
-	public User getSessionUser(HttpServletRequest request) {
+    public static final SessionService srv = new SessionService();
+
+    private static final String USER_SESSION_KEY = "loggedInUser";
```

Comment:
You replaced the user session with this key. However, you forgot to replace it in the UserDao, where the session is created. Could you use this key on line 189 in the UserDao?